src/hotspot/cpu/x86/assembler_x86.hpp

```diff
@@ -1397,135 +1397,6 @@ class Assembler : public AbstractAssembler  {
  public:
 #ifndef _LP64
   void emms();
-
```

Comment:
So you keeping this 32-bit code for now?

Reply:
Yeah, I want clean cuts. These are not FPU-related. Once we clear large chunks, leftover removals would be clearer.